scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/policy/license/LicenseSchedulingPolicy.java

```diff
@@ -186,17 +187,30 @@ private boolean isTaskExecutableUsingLicense(EligibleTaskDescriptor task) {
 
             // Do not execute if one of the required license can not be obtained
             final String[] requiredLicensesArray = requiredLicenses.split(",");
-            for (int i = 0; i < requiredLicensesArray.length; i++) {
+            final int nbRequiredLicenses = requiredLicensesArray.length;
+            boolean[] jobAlreadyUseLicense = new boolean[nbRequiredLicenses];
+
+            for (int i = 0; i < nbRequiredLicenses; i++) {
 
                 final String currentRequiredLicense = requiredLicensesArray[i];
```

Comment:
The variable initialization should be done outside de loop.

Reply:
Ok